public/app/features/alerting/unified/utils/datasource.ts

```diff
@@ -147,7 +148,11 @@ export function getAlertManagerDataSourcesByPermission(permission: 'instance' |
     silence: silencesPermissions.read,
   };
 
-  const builtinAlertmanagerPermissions = Object.values(permissions).flatMap((permissions) => permissions.grafana);
+  const builtinAlertmanagerPermissions = [
+    ...Object.values(permissions).flatMap((permissions) => permissions.grafana),
+    ...PERMISSIONS_CONTACT_POINTS,
+  ];
```

Comment:
Needed to show the Grafana dropdown option if you gave a user _only_ contact point creation permissions - as otherwise they wouldn't see the option in the dropdown and the page would be blocked from rendering by the alert saying "you don't have any AMs"

Reply:
I wonder if we shouldn't include it in the `permissions` object, but I'd need to dive a bit deeper into the  code

Reply:
Are you okay if I leave it as is for now?